front/lib/plans/trial.ts

```diff
@@ -0,0 +1,20 @@
+import type { SubscriptionType } from "@dust-tt/types";
+
+import type { Plan, Subscription } from "@app/lib/models";
+import type { PlanAttributes } from "@app/lib/plans/free_plans";
+
+// These limits are applied to all plans during the trial period.
+const TRIAL_LIMITS: Partial<PlanAttributes> = {
+  maxUsersInWorkspace: 5,
```

Comment:
We need the max user messages as well or is it not handled by the plan limits ?

Reply:
Yep it will come in another PR since today's logic for max messages does not apply to the day but the entire lifetime of the workspace.

Reply:
I'll do a cheap version that still uses the DB but IMHO, it should live in Redis.